webpack.config.js

```diff
@@ -5,6 +5,7 @@ const MomentLocalesPlugin = require("moment-locales-webpack-plugin");
 const { WebpackDeduplicationPlugin } = require("webpack-deduplication-plugin");
 module.exports = async function (env, argv) {
   const config = await createExpoWebpackConfigAsync(env, argv);
+  config.externals = ["crypto"];
```

Comment:
These changes resolve the following error; Let me know if this is the correct way to fix this error. Thanks.

```
ERROR in ./node_modules/@cosmjs/crypto/build/pbkdf2.js:14:19
Module not found: Can't resolve 'crypto'

ERROR in ./node_modules/@cosmjs/ledger-amino/node_modules/@cosmjs/crypto/build/pbkdf2.js:14:23
Module not found: Can't resolve 'crypto'
```

Reply:
it might be better to add a fallback, where did you find this solution?